packages/e2e/cypress/support/commands.ts

```diff
@@ -66,3 +66,9 @@ Cypress.Commands.add('findInputField', (field: string) => {
     return cy.findByRole('textbox', { name: regex });
   }
 });
+
+Cypress.Commands.add('waitForMapIdle', () => {
+  cy.window().should((windowObj) => {
+    expect(windowObj['mapIdle']).to.be.true;
```

Comment:
Doing our `expect` assertion within a callback to `should` tells Cypress to retry this step until a timeout which achieves our desired wait.